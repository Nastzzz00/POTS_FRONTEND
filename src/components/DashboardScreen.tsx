import React from 'react'
import { Container, Row, Col } from 'react-bootstrap'
import AdminMenu from './AdminMenu'
import UserInfo from './UserInfo'
import DashboardContent from './DashboardContent'
import { menu, statusReport, poList } from '../data/mockData'

export interface IDashboardScreenProps {}

const DashboardScreen = (props: IDashboardScreenProps) => {
  return (
    <Container>
      <Row>
        <Col md={3}>
          <AdminMenu
            title="POTS"
            desc="Purchase Order Tracking System"
            menuItem={menu}
          />
        </Col>
        <Col md={8}>
          <UserInfo user="Mark Nabablit" date="February 5, 2020" />
          <DashboardContent status={statusReport} list={poList} />
        </Col>
      </Row>
    </Container>
  )
}

export default DashboardScreen
