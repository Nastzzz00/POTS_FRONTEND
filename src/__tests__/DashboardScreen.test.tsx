/**
 * @format
 */

import React from 'react'
import { shallow } from 'enzyme'
// import ShallowRenderer from "react-test-renderer/shallow";

// // Note: test renderer must be required after react-native.
// import renderer from "react-test-renderer";

import DashboardScreen from '../components/DashboardScreen'

const props = {
  title: 'Dashboard Screen',
}

describe('Dashboard Screen', () => {
  test('renders correctly', () => {
    expect(shallow(<DashboardScreen {...props} />)).toMatchSnapshot()
  })
})
