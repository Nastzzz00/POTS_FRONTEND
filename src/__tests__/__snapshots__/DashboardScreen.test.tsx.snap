// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Screen renders correctly 1`] = `ShallowWrapper {}`;
